templates/scripts/add-vpn-user.sh

```diff
@@ -31,6 +32,21 @@ while [[ "$existing_ips" =~ "$next_ip" ]]; do
   next_ip=${next_ip%.*}.$((${next_ip##*.}+1))
 done
 
+# get DNS server setting
+dns_server=$($EXEC "cat /etc/resolv.conf | grep nameserver | tail -1 | cut -d\" \" -f2 | tr -d \"\r\n\f\"")
+
+# get VPC CIDR for allowed IP subnet
+VPCNAME=${CLUSTER%-$REGION}-vpc
+vpc_cidr=$(aws ec2 describe-vpcs --filters Name=tag:Name,Values=${VPCNAME} | jq -r '.Vpcs[].CidrBlock')
+[[ -z "$vpc_cidr" ]] && vpc_cidr = "10.10.0.0/16"
+
+# get DB server
+K8S_DBSERVER=database.${NAMESPACE}.svc.cluster.local
```

Comment:
Let's take this out. It's extra complexity just to show an example comment. Let's just leave the comment as an example like <rds mysql hostname>